slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -71,31 +79,31 @@ Slido:
 # What is RAP?
 
 ::: {.fragment .fade-up}
-<span style="color:#f67400;">**R**</span>eproducible <span style="color:#f67400;">**A**</span>nalytical <span style="color:#f67400;">**P**</span>ipelines are simply about using automation to our advantage when analysing data, and ensuring we are transparent when we do!
+<span style="color:#f67400;">**R**</span>eproducible <span style="color:#f67400;">**A**</span>nalytical <span style="color:#f67400;">**P**</span>ipelines are simply about having well-documented, reusable processes when analysing data, and ensuring we are transparent when we do!
 :::
 
 ::: {.fragment .fade-up}
 We already have *analytical pipelines* and have done for many years. The aim of RAP is to:
 :::
 
 ::: {.fragment .fade-up}
--   Automate all of the parts that we can
+-   Automate all of the parts that we can, e.g. rather than entering a hardcoded date repeatedly throughout your code, instead use one date variable that can be changed in one place and reused throughout
 :::
 
 ::: {.fragment .fade-up}
--   Increase efficiency and accuracy
+-   Increase efficiency and accuracy by reducing the risk of human error, and making sure that calculations are done the same way each time wherever possible
 :::
 
 ::: {.fragment .fade-up}
--   Create a clear audit trail to allow analysis to be easily reused if needed
+-   Create a clear audit trail using documentation to allow analysis to be easily reused where needed, either for repeated publications, or for PQ and FOI requests
```

Comment:
I think the code itself is also part of the audit trail so 'using documentation' seems the wrong phrasing to me. Maybe 'Create a clear audit trail using code and documentation to allow...'